app/scenes/Document/components/Document.js

```diff
@@ -383,28 +381,49 @@ class DocumentScene extends React.Component<Props> {
             >
               {document.isTemplate && !readOnly && (
                 <Notice muted>
-                  You’re editing a template. Highlight some text and use the{" "}
-                  <PlaceholderIcon color="currentColor" /> control to add
-                  placeholders that can be filled out when creating new
-                  documents from this template.
+                  <Trans>
+                    You’re editing a template. Highlight some text and use the{" "}
+                    <PlaceholderIcon color="currentColor" /> control to add
+                    placeholders that can be filled out when creating new
+                    documents from this template.
+                  </Trans>
                 </Notice>
               )}
               {document.archivedAt && !document.deletedAt && (
                 <Notice muted>
-                  Archived by {document.updatedBy.name}{" "}
-                  <Time dateTime={document.archivedAt} /> ago
+                  <Trans
+                    defaults="Archived by {{ userName }} <time> ago"
+                    values={{ userName: document.updatedBy.name }}
+                    components={{
+                      time: <Time dateTime={document.archivedAt} />,
+                    }}
+                  />
                 </Notice>
               )}
               {document.deletedAt && (
                 <Notice muted>
-                  Deleted by {document.updatedBy.name}{" "}
-                  <Time dateTime={document.deletedAt} /> ago
+                  <Trans
+                    defaults="Deleted by {{ userName }} <time> ago"
+                    values={{ userName: document.updatedBy.name }}
+                    components={{
+                      time: <Time dateTime={document.deletedAt} />,
+                    }}
+                  />
                   {document.permanentlyDeletedAt && (
                     <>
                       <br />
-                      This {document.noun} will be permanently deleted in{" "}
-                      <Time dateTime={document.permanentlyDeletedAt} /> unless
-                      restored.
+                      <Trans
+                        defaults="This {{ documentName }} will be permanently deleted in <time> unless restored."
+                        values={{
+                          userName: document.updatedBy.name,
+                          documentName: document.noun,
+                        }}
+                        components={{
+                          time: (
+                            <Time dateTime={document.permanentlyDeletedAt} />
+                          ),
+                        }}
+                      />
```

Comment:
{document.template ? (
                        <Trans>
                          This template will be permanently deleted in Time
                          dateTime={document.permanentlyDeletedAt} /> unless
                          restored.
                        </Trans>
                      ) : (
                        <Trans>
                          This document will be permanently deleted in Time
                          dateTime={document.permanentlyDeletedAt} /> unless
                          restored.
                        </Trans>
                      )}